src/api/hosting/web/emailOption/hosting-web-email-option.v6.service.js

```diff
@@ -0,0 +1,33 @@
+angular.module('ovh-api-services').service('OvhApiHostingWebEmailOptionV6', ($resource, $cacheFactory) => {
+  const cache = $cacheFactory('OvhApiHostingWebEmailOptionv6Cache');
+  const queryCache = $cacheFactory('OvhApiHostingWebEmailOptionv6CacheQuery');
+
+  const resource = $resource('/hosting/web/:serviceName/emailOption', {
+    serviceName: '@serviceName',
+  }, {
+    query: {
+      method: 'GET',
+      cache: queryCache,
+      isArray: true,
+    },
+    get: {
```

Comment:
What is the get used for ? There is no `emailOptionId` parameter declared in the route

Reply:
it's not used as of now, it might be useful
 in the future. I will add the id in the url

Reply:
i have added id in url